app/src/test/java/nerd/tuxmobil/fahrplan/congress/schedule/LectureViewDrawerTest.kt

```diff
@@ -0,0 +1,227 @@
+package nerd.tuxmobil.fahrplan.congress.schedule
+
+import android.widget.LinearLayout
+import info.metadude.android.eventfahrplan.commons.temporal.Moment
+import nerd.tuxmobil.fahrplan.congress.models.Lecture
+import nerd.tuxmobil.fahrplan.congress.schedule.LectureViewDrawer.Companion.DIVISOR
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+class LectureViewDrawerTest {
+    private val standardHeight = 1
+    private val conference = Conference()
+    private val conferenceDate = "2020-03-30"
+    private var lectureId = 0
+
+    private fun createLecture(room: Int, date: String? = null, time: Int = 0, _duration: Int = 0): Lecture {
```

Comment:
Please rename `room` to `roomIndex` to avoid confusion.

Reply:
Please do not start variable names with `_`. You can use `this.roomIndex` within the `apply` block to avoid this.

Reply:
Resolved in 39f5879d46110c37c99f54af2828a2feb9ab82ba. Thank you.